<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fontGroupBox">
     <property name="title">
      <string>Font Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fontFamilyLabel">
        <property name="text">
         <string>Font Family:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fontSizeLabel">
        <property name="text">
         <string>Font Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="fontSizeSpinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="languageGroupBox">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Interface Language:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="languageCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string>Display Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showArabicCheckBox">
        <property name="text">
         <string>Show Arabic Text</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showLatinCheckBox">
        <property name="text">
         <string>Show Latin Transliteration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="highlightColorLabel">
          <property name="text">
           <string>Highlight Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="highlightColorButton">
          <property name="text">
           <string></string>
          </property>
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
